src/johnsession.cpp

```diff
@@ -173,7 +175,7 @@ bool JohnSession::save()
 
     // Advanced options
     if (isForkEnabled()) {
-        m_settings->setValue("nbForkProcess", m_nbProcess);
+        m_settings->setValue("nbForkProcess", m_fork);
```

Comment:
I took extra care of supporting any session made with johnny 2.0 by not changing the settings key. Do you really want to loose that ?